### DISCUSSION TITLE
使用accessToken访问 `/api/me` 失败

### DISCUSSION THREAD

**Discussion by JZLeung** [logto-io/logto#1802] [Q&A]
使用 `const { getAccessToken } = useLogto()` 获取 `access_token`，再使用该 token 去访问接口 `/api/me` 会报错：
```js
var myHeaders = new Headers();
myHeaders.append("Authorization", "access_token from getAccessToken()");

var requestOptions = {
  method: 'GET',
  headers: myHeaders,
  redirect: 'follow'
};

fetch("https://logto.example.com/api/me", requestOptions)
  .then(response => response.text())
  .then(result => console.log(result))
  .catch(error => console.log('error', error));
```
返回：
```json
{
    "message": "未经授权。请检查凭据及其范围。",
    "code": "auth.unauthorized"
}
```
如果访问 `/api/session` 会返回：
```json
{
    "message": "未找到会话。请返回并重新登录。",
    "code": "session.not_found",
    "data": {
        "error_description": "interaction session id cookie not found"
    }
}
```

**Comment by gao-sun** [comment]
from our [issue template](https://raw.githubusercontent.com/logto-io/.github/master/.github/ISSUE_TEMPLATE/bug_report.md):

>For non-English users: It's okay to post in your language, but remember to use English for the body (you can paste the result of Google Translate), and put everything else as attachments.
>
> Issues with a non-English body will be DIRECTLY CLOSED until it's updated.

this also applies to our discussion. would you like to update the body if convenient? thanks.
